speech/unit_tests/test_client.py

```diff
@@ -192,45 +214,48 @@ def test_sync_recognize_with_empty_results_no_gax(self):
 
     def test_sync_recognize_with_empty_results_gax(self):
         from google.cloud._testing import _Monkey
-        from google.cloud.speech import _gax as MUT
+
         from google.cloud import speech
+        from google.cloud.speech import _gax as MUT
```

Comment:
How about....
```diff
- MUT  # Module under test
+ MUM  # Module under mock
```

I guess the real point here is that I need to mock out `SpeechApi`, I guess I could try and override it through `client`.